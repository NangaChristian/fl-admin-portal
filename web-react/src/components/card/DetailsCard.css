.detail-card {
  margin-top: 10px;
  margin-bottom: 10px;
  width: 100%;
}

.bg-card {
  background-color: var(--card);
  padding: 10px;
  border-radius: 10px;
}

.card-heading {
  font-size: 14px;
}

.card-detail {
  margin-bottom: 0px;
  font-size: 22px;
}
