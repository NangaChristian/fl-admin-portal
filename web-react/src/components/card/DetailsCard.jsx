import React from 'react'
import './DetailsCard.css'

const DetailsCard = (props) => {
  return (
    <div>
      <div className="container-fluid card detail-card m-3">
        <div className="row">
          <div className="col-auto">
            <span className="text-secondary">{props.heading}</span>
            <h2 className="font-weight-bold">{props.detail}</h2>
          </div>
          {props.heading2 && (
            <div className="col-auto">
              <span className="text-secondary">{props.heading2}</span>
              <h2 className="font-weight-bold">{props.detail2}</h2>
            </div>
          )}
        </div>
      </div>
    </div>
  )
}

export default DetailsCard
