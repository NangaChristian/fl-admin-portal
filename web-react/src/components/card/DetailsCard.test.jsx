import React from 'react'
import { render, cleanup } from '@testing-library/react'
import '@testing-library/jest-dom/extend-expect'
import DetailsCard from './DetailsCard'

afterEach(cleanup)

it('renders', () => {
  const { asFragment } = render(<DetailsCard>Heading</DetailsCard>)
  expect(asFragment()).toMatchSnapshot()
})
