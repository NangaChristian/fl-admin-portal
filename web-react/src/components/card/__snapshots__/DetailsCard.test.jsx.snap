// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<DocumentFragment>
  <div>
    <div
      class="container-fluid card detail-card m-3"
    >
      <span
        class="text-secondary"
      />
      <h2
        class="font-weight-bold"
      />
    </div>
  </div>
</DocumentFragment>
`;
